src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -718,3 +726,27 @@
              {:current-screen stack-id
               :start-flow?    start-flow?
               :flow-id        flow-id}]]]})))
+
+(rf/reg-event-fx
+ :wallet/transaction-confirmation-navigate-back
+ (fn [{db :db} [{:keys []}]]
```

Comment:
Empty destructuring?

Reply:
I'll remove it :+1:  thanks for noticing :+1: